modules/nf-core/nanoqc/main.nf

```diff
@@ -0,0 +1,52 @@
+process NANOQC {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanoqc:0.10.0--pyhdfd78af_0':
+        'biocontainers/nanoqc:0.10.0--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fastq)
+
+    output:
+    tuple val(meta), path("*.html"), emit: report
+    tuple val(meta), path("*.log") , emit: log
+    path "versions.yml"            , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args   = task.ext.args   ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    nanoQC \\
+        $fastq \\
+        $args
+
+    mv *.html ${prefix}.html
+    mv *.log ${prefix}.log
```

Comment:
Can you do it the more explicit way and match the file name directly without the globing?
Currently it looks to me like you're trying to match multiple files and rename them to the same.
Assuming the outputs are files and `${prefix}.html` is no directory 😄

Reply:
Looks like the tool always makes files called NanoQC.log and NanoQC.html, so be good to use those names as Leon says.